terraform/modules/configs/deploy-all/main.tf

```diff
@@ -405,6 +405,14 @@ resource "aws_iam_role_policy_attachment" "ecs_task_execution_role_read_ssm" {
 # Modules
 ######################################
 
+module "lambda" {
```

Comment:
Could make this module name a bit clearer - `s3_virus_scan_lambda` or perhaps better still, bundle it into the `s3-virus-scan` service directory so the two closely related components are collocated?